switch.tscn

```diff
@@ -0,0 +1,102 @@
+[gd_scene load_steps=18 format=3 uid="uid://dm12uriit1jpb"]
+
+[ext_resource type="Texture2D" path="res://assets/switch.png.png" id="1_gec7q"]
+[ext_resource type="Script" path="res://addons/block_code/block_code_node/block_code.gd" id="2_eobrb"]
+[ext_resource type="Script" path="res://addons/block_code/serialization/block_serialization_tree.gd" id="3_o6qre"]
```

Comment:
This scene won't load because you (correctly) removed the block code plugin from this PR – I think this scene should be removed from this PR too.